src/target/riscv/riscv-013.c

```diff
@@ -236,11 +237,13 @@ static dm013_info_t *get_dm(struct target *target)
 		return info->dm;
 
 	int abs_chain_position = target->tap->abs_chain_position;
+	uint32_t dbgbase = target->dbgbase;
```

Comment:
I'd remove the local `dbgbase` and just refer to target->dbgbase directly where it's needed.

Reply:
done